### PR TITLE
Add support for python3.7

### PR DESCRIPTION
**Pull request by Zempashi**
Some linux distribution (Archlinux ;) ) has python3.7 landed. Installation of pyyaml 3.12 broke.

```
Creating virtualenv...
/home/julien/repos/Zenko-Private-Tools/metal-k8s/.shell-env/virtualenv-15.2.0/virtualenv.py:1047: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alter
native uses
  import imp
Installing Python dependencies...
  Failed building wheel for pyyaml
Command "/home/julien/repos/Zenko-Private-Tools/metal-k8s/.shell-env/metal-k8s/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-vrgaiv_6/pyyaml/setup.py';f=getattr(tokenize, 'open', open)
(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-ud7gis6x/install-record.txt --single-version-externally-managed --compile --instal
l-headers /home/julien/repos/Zenko-Private-Tools/metal-k8s/.shell-env/metal-k8s/include/site/python3.7/pyyaml" failed with error code 1 in /tmp/pip-install-vrgaiv_6/pyyaml/                                      
```

This is solved in 3.13.

### CODE REVIEW

**Comment by NicolasT** [comment]
Should this go in 1.0 instead?

File: ChangeLog.rst [hunk 1]
```diff
@@ -42,6 +42,8 @@ Kubespray
 
 :ghpull:`223` - pre-provision Kibana index configuration (:ghissue:`174`)
 
+:ghpull:`274` - Add support for python3.7
```
Comment (NicolasT) [inline ChangeLog.rst:45]: Nit-pick, but: for consistency, no capital here.

**Review by NicolasT — APPROVED** [review]

**Comment by Zempashi** [comment]
Not handled by bert-e, because of wrong branch naming

File: ChangeLog.rst (hunk 1 above)
Comment (Zempashi) [reply ChangeLog.rst:45]: Oups and added in wrong section

**Pull request closed without merging**